tests/test_simulator.py

```
@@ -35,6 +37,7 @@
 )
 from mqt.yaqs.core.libraries.circuit_library import create_ising_circuit
 from mqt.yaqs.core.libraries.gate_library import XX, YY, ZZ, X, Z
+from mqt.yaqs.simulator import _get_parallel_context, _worker_init  # noqa: PLC2701
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)